dhis-2/dhis-support/dhis-support-commons/src/main/java/org/hisp/dhis/commons/util/SystemUtils.java

```diff
@@ -51,6 +51,11 @@ public static boolean isEnableCacheInTest( String[] profiles )
         return Arrays.asList( profiles ).contains( "cache-test" );
     }
 
+    public static boolean isCacheInvalidationInTest( String[] profiles )
+    {
+        return Arrays.asList( profiles ).contains( "cache-invalidation-test" );
```

Comment:
Usually `Arrays.asList` can not be replaced by `List.of` unless you worry that the profiles array might include a `null` value.